Kickstarter-iOS/Views/Cells/DiscoveryOnboardingCell.swift

```diff
@@ -62,6 +62,7 @@ private let discoveryOnboardingTitleStyle: LabelStyle = { label in
 private let discoveryOnboardingLogoStyle: ImageViewStyle = { imageView in
   imageView
     |> \.contentMode .~ .scaleAspectFit
+    |> \.tintColor .~ .ksr_green_500
```

Comment:
Once rebased this can change to `ksr_create_700`.

Reply:
Merged the changes from your work in `NT-1090` and updated some snapshots after resolving merge conflicts. Is there an existing feature branch to change the base of this PR?